pages/index.js

```diff
@@ -13,10 +13,19 @@ export default function Home() {
     setItems(items.filter((item) => item.id !== id));
   }
 
+  function handleEditItem(editItem) {
```

Comment:
```suggestion
  function handleEditItem(updatedItem) {
```
As naming suggestion in ItemCard.js